Assets/AdditionalScripts/Start Screen Scripts/StartButtonScript.cs

```diff
@@ -5,9 +5,12 @@
 
 public class StartButtonScript : MonoBehaviour
 {
-    public void LoadScene(string sceneName)
+    public void LoadScene()
```

Comment:
```suggestion
    public void LoadGame()
```
method should be called LoadGame() like it is in the PlayAgainButtonScript

Reply:
true, ty